### DISCUSSION TITLE
Open up for group discussion: Radiotherapy

### DISCUSSION THREAD

**Discussion by rtmill** [OHDSI/OncologyWG#488] [Ideas]
# Problem Space

Create a standard representation for radiotherapy data. Specifically, what level of granularity for procedures and which details should be recorded as modifiers. 

## Factors
- Type
- Phase / course 
- Technique
- Site
- Modality
- Intent
- Fractions
- Dose/Volume
- Planned vs. Delivered
- Start date and end date (if procedures) 
- Others?

# Proposed Solutions

We need to identify what level of granularity the actual "procedure" of the radiotherapy will be and consequently what modifiers will be needed to fully represent the relevant details. 

Assuming we want to have the standard target concept (procedure domain) be more specific than "radiotherapy" with a bunch of modifiers, is there a hierarchy of procedures that we can leverage? 

One suggestion has been to reference the "Astro Minimum Data Elements for Radiation Oncology". An article on this can be found here: https://www.practicalradonc.org/article/S1879-8500(19)30232-2/fulltext

Another factor is that all of the above is referencing data sources that contain detailed radiotherapy data but we also need to account for the more generic radiotherapy procedures which lack this detail, e.g. from billing codes alone, and how we can accommodate both types in a similar manner to enable straightforward phenotyping.

**Comment by peterprinsen-iknl** [comment]
For the Netherlands Cancer Registry we currently map radiotherapy on SNOMED but it is hard to find matching concepts so we usually map to something more general. Finding a matching modality is usually not the problem. We do not register radiotherapy in great detail so proton therapy/external beam radiotherapy (photons)/brachytherapy/etc. work as concepts for us. The issue is the location. Unless otherwise specified this is the primary tumor. If the therapy is on the metastases then this is specified in the specific treatment (either generally as "for metastases" or specifically as "for liver metastases", "for brain metastases", etc.).

So, a modifier that specifies site would make sense in our case but, as in the surgery discussion, the question is whether this should be site or something like primary tumor, regional lymph nodes, or metastases. In the latter case it may be good to have a hierarchy (either general "metastases" or children like "liver metastases", "brain metastases", etc.).

Other data we collect for radiotherapy is start/end date and complications (during therapy or within 90 days after end of therapy).

**Comment by cgreich** [comment]
@peterprinsen-iknl:

> site or something like primary tumor, regional lymph nodes, or metastases. In the latter case it may be good to have a hierarchy (either general "metastases" or children like "liver metastases", "brain metastases", etc.).

Makes sense. 

> for radiotherapy is start/end date

Do you have cycle information?

> complications

Why would they not be normal Condition records?

**Reply by peterprinsen-iknl** [reply]
@cgreich:

> Do you have cycle information?

We have cycle information (number of cycles) and total dose but only for certain tumor types and certain time periods. I haven't looked into the details but for some tumor types it seems we started registering number of cycles a few years ago while for others we did it for a few years and then stopped registering it. We haven't added this information to our OMOP-CDM (yet).

> Why would they not be normal Condition records?

We link it to the treatment (as a detail) in our cancer registry but it makes sense to create a normal Condition record in OMOP.
